src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

```diff
@@ -82,15 +82,22 @@ Remember that the user type governs a user's theoretical access. In practice, us
 
 Want to understand how billing works? See [User billing calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing).
 
-## Detailed capability comparison table [#user-type-comparison-table]
+## Detailed user type comparison table [#user-type-comparison-table]
 
-Here's a detailed comparison of capabilities by user type. For tips on why you'd make a user one user type or another, see [Decide on user type](#choose-user-type).
+Here's a detailed comparison of capabilities by user type. 
+
+Important points about this table: 
+* "Capabilities" in this context refers to high-level New Relic features and functionality. 
+* User type is how you control user access based on that user's long term responsibilities. The more granular way to control access is using roles. [Learn more about controlling user access.](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#user-type-groups-relation)
+* The capabilities in the table below, like APM and browser monitoring, refer to access to UI experiences. All users, including basic users, are able to query all data in the accounts they have access to, and create and view custom charts. 
```

Comment:
@zuluecho9 This first sentence made me stumble a bit. How about something more like this: The capabilities in this table refer to your access to New Relic UI experiences, such as for APM and browser monitoring.

For the second sentence, how about a revision similar to this: While all user types don't have access to every feature, all user types , including basic users, can query data and create charts for accounts they have access to.